### PR TITLE
Increased window position precision to 1/16th of a pixel

### PR DESCRIPTION
**Pull request by Gaspard--**
I can't change window size precision, but did change container precision size too internally.

### CODE REVIEW

File: include/View.hpp [hunk 1]
```diff
@@ -46,7 +47,7 @@ class View : public ViewListeners
   Server *server;
   struct wlr_xdg_surface_v6 *xdg_surface;
   bool mapped;
-  int x, y;
+  FixedPoint<-4, int> x, y;
```
Comment (riamon-v) [inline include/View.hpp:50]: What does all the "-4" means ? Is it like the offset for the window to be placed properly ? (aka the window borders)

Comment (riamon-v) [reply include/View.hpp:50]: Otherwise everything looks good to me at the first sight

**Review by riamon-v — APPROVED** [review]
LGTM

File: include/View.hpp (hunk 1 above)
Comment (Gaspard--) [reply include/View.hpp:50]: -4 means 2^-4 precision, aka 1/16th.

**Pull request merged**